flags.go

```diff
@@ -341,6 +345,69 @@ type flagStorage struct {
 	DebugMutex      bool
 }
 
+const GCSFUSE_PARENT_PROCESS_DIR = "gcsfuse-parent-process-dir"
+
+// 1. Returns the same filepath in case of absolute path or empty filename.
+// 2. For relative path with . or .. or nothing, it resolves smartly the path
+// in case of child proces it resolves with respect to GCSFUSE_PARENT_PROCESS_DIR
+// otherwise with respect to the current working dir.
+// 3. For relative path starting with ~, it resolves with respect to home dir.
+func getResolvedPath(filePath string) (resolvedPath string, err error) {
+	if filePath == "" || path.IsAbs(filePath) {
+		resolvedPath = filePath
+		return
+	}
+
+	// Relative path starting with tilda (~)
+	if strings.HasPrefix(filePath, "~/") {
+		homeDir, err := os.UserHomeDir()
+		if err != nil {
+			return "", fmt.Errorf("Fetch home dir: [%w]", err)
+		}
+		return filepath.Join(homeDir, filePath[2:]), err
+	}
+
+	// Means - relative path starting with ., .. or nothing.
```

Comment:
instead of means mention - we reach here when relative path starts with . or .. or nothing